### PR TITLE
[#20] error message formatting

### PR DESCRIPTION
**Pull request by pavertomato**

### CODE REVIEW

File: src/main/java/com/google/dicomwebfuse/dao/FuseDaoImpl.java [hunk 1]
```diff
@@ -382,9 +386,11 @@ private void createRequestForUploadInstance(URIBuilder uriBuilder, Path instance
       try (CloseableHttpResponse response = httpclient.execute(request)) {
         int statusCode = response.getStatusLine().getStatusCode();
         if (statusCode != HttpStatusCodes.STATUS_CODE_OK) {
+          Document parsedXmlResponse = Jsoup
+              .parse(EntityUtils.toString(response.getEntity()), "", Parser.xmlParser());
```
Comment (dzelemba) [inline src/main/java/com/google/dicomwebfuse/dao/FuseDaoImpl.java]: So unfortunately not all responses will be in XML format... Anything that gets to the DICOMweb portion will be XML, but anything at the Healthcare API portion (e.g. permission denied, missing dicom store) will be JSON. Can we handle that here and add a unit tests for both of these cases?

Comment (pavertomato) [reply src/main/java/com/google/dicomwebfuse/dao/FuseDaoImpl.java]: should json be formatted as well?

Comment (dzelemba) [reply src/main/java/com/google/dicomwebfuse/dao/FuseDaoImpl.java]: I believe the JSON is already nicely formatted.

Comment (pavertomato) [reply src/main/java/com/google/dicomwebfuse/dao/FuseDaoImpl.java]: done

**Pull request merged**